### PR TITLE
Eepromutils cleanup

### PR DESCRIPTION
**Pull request by iluminat23**
Mostly cosmetic changes.
One fix for eepflash.sh.
And a small change to the usage output of eepflash.sh

### CODE REVIEW

File: eepromutils/eeptypes.h [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
-#define ATOM_GPIO_TYPE	0x0002
+#define ATOM_INVALID_TYPE	0x0000
+#define ATOM_VENDOR_TYPE	0x0001
+#define ATOM_GPIO_TYPE		0x0002
 #define ATOM_DT_TYPE 		0x0003
-#define ATOM_CUSTOM_TYPE		0x0004
+#define ATOM_CUSTOM_TYPE	0x0004
 #define ATOM_HINVALID_TYPE	0xffff
 
-#define ATOM_VENDOR_NUM 0x0000
```
Comment (pelwell) [inline eepromutils/eeptypes.h:11]: This line and the next end up being mis-indented - see https://github.com/raspberrypi/hats/blob/21abd81cf03cc72b17aaeec05e3eba87ad4cffd6/eepromutils/eeptypes.h#L11.

File: eepromutils/eeptypes.h [hunk 2]
```diff
@@ -70,53 +70,53 @@ struct gpio_map_d {
 
 
 uint16_t getcrc(char* data, unsigned int size) {
-	
+
```
Comment (pelwell) [inline eepromutils/eeptypes.h:73]: Delete this blank line completely.

File: eepromutils/eepflash.sh [hunk 3]
```diff
@@ -33,13 +33,13 @@ usage()
...
-   exit 1
+	echo "This script must be run as root" 1>&2
+	exit 1
 fi
- 
+
 while [ "$1" != "" ]; do
-    PARAM=`echo $1 | awk -F= '{print $1}'`
-    VALUE=`echo $1 | awk -F= '{print $2}'`
+	PARAM=$(echo $1 | awk -F= '{print $1}')
```
Comment (pelwell) [inline eepromutils/eepflash.sh:43]: Although I can see where you are coming from when you say this change is cosmetic, I think the backtick-to-`$(` translation deserves more attention than the commit message suggests - either mention it explicitly or break the change out into another commit.

Comment (iluminat23) [reply eepromutils/eepflash.sh:43]: Oooohh. Sorrz this change was not supposed to be included. Just did it to please the shellcheck script. And then I have overseen it.

File: eepromutils/eeptypes.h (hunk 2 above)
Comment (iluminat23) [reply eepromutils/eeptypes.h:73]: Ack.

File: eepromutils/eeptypes.h (hunk 1 above)
Comment (iluminat23) [reply eepromutils/eeptypes.h:11]: This depends on the tab-width. I assumed 4 spaces per tab as the other code was indented with 4 spaces per indention level. If someone would make a decision about the indention I will change it.

Comment (pelwell) [reply eepromutils/eeptypes.h:11]: Most Linux-based editors (emacs, vi, nano, ...) and the GitHub viewer set the TAB width to be 8, and that is what most open-source software (including the Linux kernel) expects. For non-kernel work I'm happy with 4-character indents, and for that to be consistently _either_ all spaces _or_ a mixture of 8-character TABs and groups of 4 spaces, with spaces never appearing before TABs.

You can completely sidestep the problem of mis-alignment due to TAB sizes by only using TABs at the start of a line, with any mid-content alignment being done with space - perhaps this is the solution.

**Comment by pelwell** [comment]
Thanks!

**Pull request merged**